modules/evm/src/runner/handler.rs

```diff
@@ -49,6 +49,8 @@ pub struct Handler<'vicinity, 'config, 'meter, T: Config> {
 	pub gasometer: Gasometer<'config>,
 	pub storage_meter: StorageMeter<'meter>,
 	pub is_static: bool,
+	/// precompile return error message.
+	pub err_msg: Option<Vec<u8>>,
```

Comment:
It needs to be cleared at some stage to ensure we don't accidentally return error message from previous error